source/includes/linux/common-installation.rst

```diff
@@ -233,6 +233,11 @@ file manually on all MinIO hosts:
    ExecStartPre=/bin/bash -c "if [ -z \"${MINIO_VOLUMES}\" ]; then echo \"Variable MINIO_VOLUMES not set in /etc/default/minio\"; exit 1; fi"
    ExecStart=/usr/local/bin/minio server $MINIO_OPTS $MINIO_VOLUMES
 
+   # MinIO RELEASE.2023-05-04T21-44-30Z adds support for Type=notify (https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=)
+   # This may improve systemctl setups where other services use `After=minio.server`
+   # Uncommon the line to enable the functionality
```

Comment:
`uncomment` I presume. Is this example from a code repo or somewhere else that might further propagate a typo?

Reply:
No this is from me :grimacing: